subprojects/docs/src/docs/userguide/dependencyManagementTerminology.adoc

```diff
@@ -0,0 +1,44 @@
+[[dependency_management_terminology]]
+== Dependency Management Terminology
+
+Dependency management comes with a wealth of terminology. Here you can find the most commonly-used terms including references to the user guide to learn about their practical application.
+
+[[sub:terminology_module]]
+Module::
+
+A piece of software that evolves over time e.g. link:https://github.com/google/guava[Google Guava]. Every module has a name. Each release of a module is optimally represented by a <<sub:terminology_version,version>>. For convenient consumption, modules can be hosted in a <<sub:terminology_repository,repository>>.
+
+[[sub:terminology_version]]
+Version::
+
```

Comment:
What's a "revision". 
What's an example of a "version"? 
What is a "module version"?
Is the 1.0 release of module "org:foo" a "version"? If not, what do we call it?

I think this terminology is confusing, and this description doesn't make it clearer. More examples might help.

Reply:
I am OK with calling it a "module version" as it clearly indicates that it belongs to a module. How about the following? 

"A module version represents a distinct set of changes of a released module. For example `18.0` is the module version of the module with the corrdinates `com.google:guava:18.0`."